tensorly/datasets/data_imports.py

```diff
@@ -853,16 +853,14 @@ def load_indian_pines():
     )
 
 
-def fetch_kinetic():
+def load_kinetic():
     """
-    Loads kinetic fluorescence dataset from website and returns it as tensorly tensor without storing the data
-    in the hard drive.The data is well suited for Parafac and multi-way partial least squares regression (N-PLS).
+    Loads the kinetic fluorescence dataset (X60t) as a tensorly tensor. The data is well suited for Parafac and multi-way partial least squares regression (N-PLS). Missing data are replaced by 0s, and a missing value mask is provided.
+    Data is a courtesy of Rasmus Bro and collaborators, it can be originally downloaded at https://ucphchemometrics.com/. Please cite the original reference if you use this data in any way.
```

Comment:
Can we add the reference in the doctoring as we do for e.g. CP so it appears in the API?